tests/system/action/meeting/test_import.py

```diff
@@ -351,10 +302,6 @@ def get_user_data(self, obj_id: int, data: Dict[str, Any] = {}) -> Dict[str, Any
             "id": obj_id,
             "password": "",
```

Comment:
Also remove this and all other empty strings. Same goes for all the other helper methods, there are still some fields left.

Reply:
Some of the empty strings are needed, like password.

Reply:
But not all of them, something like `title` or `pronoun` should not be required. Also, for e.g. password, we should also note in the models file that it is required.

Reply:
Removed not needed (user) fields like `title`, `pronoun` and `password`.

Reply:
Removed not needed fields like `title`,`pronoun` and `password`.